pachyderm/templates/pachd/deployment.yaml

```diff
@@ -30,16 +30,16 @@ spec:
         - /pachd
         env:
         - name: PACH_ROOT
-          value: /pach
+          value: "/pach"
         - name: ETCD_PREFIX
           #value:
         - name: NUM_SHARDS
-          value: {{ .Values.pachd.numShards | quote }}
+          value: {{ required "number of shards missing" .Values.pachd.numShards | quote }}
         - name: STORAGE_BACKEND
-          value: {{ .Values.pachd.storage.backend }}
+          value: {{ required "storage backend missing" .Values.pachd.storage.backend | quote }}
         - name: STORAGE_HOST_PATH
           {{- if eq .Values.pachd.storage.backend "LOCAL" }}
-          value: /var/pachyderm/pachd
+          value: {{ .Values.pachd.storage.local.hostPath }}pachd
```

Comment:
How come pachd is separte here?

Reply:
This is a relic of how `pachctl deploy` works: https://github.com/pachyderm/pachyderm/blob/1.13.x/src/server/pkg/deploy/assets/assets.go#L625

I would be very happy to change it, if you’re willing to break drop-in compatibility a tad more!

Reply:
This is good